user_interface/author_ui.rb

```diff
@@ -0,0 +1,41 @@
+require_relative './game_ui'
+require_relative '../modules/author'
+
+class AuthorUserInterface
+  FILE_LOCATION = './data/authors.json'.freeze
+
+  def initialize
+    @authors = load(GameUserInterface.new.create_instance_of_game)
+  end
+
+  def load(items)
+    return [] if File.size(FILE_LOCATION).zero?
+
+    data = JSON.parse(File.read(FILE_LOCATION))
+    authors = []
+    data.each do |author|
+      new_author = Author.new(author['first_name'], author['last_name'])
+      authors << new_author
+
+      author['items'].each do |_author_id|
+        item = items.find { |element| element.id == item_id }
+        author.add_item(item) unless item.nil?
+      end
+    end
+    authors
+  end
+
+  def list_authors
+    @authors.each_with_index do |author, index|
+      puts "#{index + 1}) Author Name: #{author.first_name} #{author.last_name}"
+    end
+  end
+
+  def save(records)
+    serialized = []
+    records.each do |item|
+      serialized << item.to_json
+    end
+    File.write(FILE_LOCATION, JSON.dump(serialized))
```

Comment:
- [ ] it would be better if the app doesn't crash when the `json` file is missing you should  handle that case and create a json file if the file is missing 
<img width="1123" alt="Screenshot 2023-02-17 at 16 21 39" src="https://user-images.githubusercontent.com/61790587/219695549-642959e9-f0f9-418d-8a05-bea7692c684c.png">